framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -1124,7 +1122,7 @@ private static BoundType getTypeVarBoundType(
     }
 
     /**
-     * @return the boundType (UPPER, UNBOUND, or UNKNOWN) of the declaration of typeParamElem
+     * @return the boundType (UPPER, UNBOUNDED, or UNKNOWN) of the declaration of typeParamElem
```

Comment:
UNKNOWN doesn't exist any more, so all occurrences of it should be removed from the Javadoc.